React/Views/RCTSlider.m

```diff
@@ -14,6 +14,8 @@ @implementation RCTSlider
   float _unclippedValue;
 }
 
+@synthesize step;
```

Comment:
This isn't needed. Property synthesis is automatic in most cases, and Xcode will warn you when it isn't.